src/plugin.js

```diff
@@ -143,7 +143,7 @@ const initPlugin = function(player, options) {
       let currentTime = player.currentTime();
 
       // playback is operating normally or has recovered
-      if (currentTime !== lastTime) {
+      if ((lastTime + 1) <= (currentTime)) {
```

Comment:
Previously any small time changes while a player was stalled would cause `resetMonitor` to be called. This was a problem when an initial timeout error happens, the player pauses, and this function is triggered again and the `currentTime` has changed slightly (~0.2). I don't think that we need to count time changes of less than one second in this check at all, since timeouts take 45seconds by default.